src/app/frontend/pod/detail/containerinfo_component.js

```diff
@@ -39,6 +39,9 @@ export default class ContainerInfoController {
     /** @export {string} Initialized from a binding. */
     this.namespace;
 
+    /** @export {string} Initialized from a binding. */
+    this.isInitContainer;
```

Comment:
this variable seems not to be used in the current implementation with transclude